packages/blockchain-link/package.json

```diff
@@ -19,28 +19,45 @@
         "blockbook",
         "javascript"
     ],
-    "main": "./lib/index.js",
     "files": [
         "lib/",
         "!lib/ui",
         "!**/*.map"
     ],
-    "types": "lib/index.d.ts",
-    "typings": "lib/index.d.ts",
+    "main": "src/index.ts",
     "browser": {
-        "socks-proxy-agent": "./lib/utils/socks-proxy-agent.js",
-        "ws": "./lib/utils/ws.js"
+        "socks-proxy-agent": "./src/utils/socks-proxy-agent.ts",
+        "ws": "./src/utils/ws.ts"
     },
     "react-native": {
         "__comment__": "Hotfix for issue where RN metro bundler resolve relatives paths wrong",
-        "ws": "@trezor/blockchain-link/lib/utils/ws-native.js",
-        "socks-proxy-agent": "@trezor/blockchain-link/lib/utils/socks-proxy-agent.js"
+        "ws": "@trezor/blockchain-link/src/utils/ws-native.ts",
+        "socks-proxy-agent": "@trezor/blockchain-link/src/utils/socks-proxy-agent.ts"
+    },
+    "publishConfig": {
```

Comment:
what is the relastion between 
`files` and `publishConfig.files`?

Reply:
fixed